docs/src/pages/_app.page.tsx

```diff
@@ -1,5 +1,4 @@
 import Head from 'next/head';
-import '@aws-amplify/ui/dist/style.css';
```

Comment:
This isn't needed because the theme is exported straight from `@aws-amplify/ui-react/styles.css`